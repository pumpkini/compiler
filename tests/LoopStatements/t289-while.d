
    int main(){
        while(itob(ReadInteger())){
            Print("hey");
            while (itob(ReadInteger())){
                Print("huy");
            }
        }
        while(itob(ReadInteger())){
            Print("hoy");
        }
    }
