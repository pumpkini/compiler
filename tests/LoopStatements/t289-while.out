hey
huy
huy
huy
huy
huy
hey
huy
hoy
hoy
hoy
